packages/vscode-extension/lib/network/interceptors/PolyfillFetchInterceptor.ts

```diff
@@ -0,0 +1,433 @@
+import { AsyncBoundedResponseBuffer } from "../AsyncBoundedResponseBuffer";
+import {
+  getFetchResponseDataPromise,
+  deserializeRequestData,
+  trimContentType,
+} from "../networkRequestParsers";
+import type { NetworkProxy, NativeResponseType, BlobLikeResponse } from "../types";
+
+let Fetch: any = undefined;
+try {
+  Fetch = require("react-native-fetch-api/src/Fetch")?.default;
```

Comment:
historically, we had issues with require calls inside try/catch. They'd cause bundle issues on projects where the package isn't installed. 
If this turns out to be the case here we could use the babel transformer as a way of "injecting" the overrides. 

Maybe @filip131311 remembers how did those require calls break